doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -271,9 +274,9 @@ These rows are removed and commented out as it's not clear what the requirement
 <a name="2.3"></a>
 ## 2.3 Kubernetes Architecture Requirements
 
-The Reference Model (RM) defines the Cloud Infrastructure to consist of the physical resources, virtualised resources and a software management system.  In the virtualised world, the Cloud Infrastructure consists of the Guest Operating System, Hypervisor and, if needed, other software such as libvirt.  And the Cloud Infrastructure Management is responsible for, among others, tenant management, resources management, inventory, scheduling, and access management.
+The Reference Model (RM) defines the Cloud Infrastructure, which consist of the physical resources, virtualised resources and a software management system.  In the virtualised world, the Cloud Infrastructure consists of the Guest Operating System, Hypervisor and, if needed, other software such as libvirt.  The Cloud Infrastructure Management component is responsible for, among others, tenant management, resources management, inventory, scheduling, and access management.
 
-Now consider, the containerisation equivalent. Reference to "Architecture" in this chapter refers to the Cloud Infrastructure Hardware (e.g. physical resources), Cloud Infrastructure Software (e.g. Hypervisor (optional), Container Runtime, virtual or container Orchestrator(s), Operating System), and infrastructure resources consumed by virtual machines or containers.
+Compare the RM with the containerisation equivalent, references to "Architecture" in this chapter refer to the Cloud Infrastructure Hardware (e.g. physical resources), Cloud Infrastructure Software (e.g. Hypervisor (optional), Container Runtime, virtual or container Orchestrator(s), Operating System), and infrastructure resources consumed by virtual machines or containers.
```

Comment:
`Compare the RM with ` doesn't make sense here.  The "now consider" was to consider the "In the virtualised world...." part, not the "The Reference Model....".

Reply:
@tomkivlin returned as origional